avocado_vt/plugins/vt_joblock.py

```diff
@@ -123,8 +106,9 @@ def pre(self, job):
                 self._lock(job)
         except Exception as detail:
             msg = "Failure trying to set Avocado-VT job lock: %s" % detail
-            self._abort(msg, job)
+            self.log.error(msg)
+            sys.exit(exit_codes.AVOCADO_JOB_FAIL | job.exitcode)
 
     def post(self, job):
         if self.lock_file is not None:
-            self._unlock()
+            os.unlink(self.lock_file)
```

Comment:
I created this card to keep track of a possible issue here:

https://trello.com/c/iyKAwj35/740-check-at-postjob-plugins-are-called-on-various-error-conditions